test/e2e/cluster_size_autoscaling.go

```diff
@@ -34,7 +34,7 @@ const (
 	scaleDownTimeout = 30 * time.Minute
 )
 
-var _ = Describe("Autoscaling", func() {
+var _ = Describe("[Autoscaling] [Skipped]", func() {
```

Comment:
The `[Autoscaling]` tag used in other tests, not just this describe block. So I wanted to be consistent in case people filter with the brackets. It's a subtle difference between a group tag used for filtering and a describe/context block just used for abstraction.